### PR TITLE
add jekmdl

### PR DESCRIPTION
**Pull request by tigefa4u**
adding jekmdl themes :+1:

### CODE REVIEW

File: _posts/2016-01-01-jekmdl.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Jekmdl"
```
Comment (gynter) [inline _posts/2016-01-01-jekmdl.markdown]: Hey and thanks for the PR! Please remove the excess quotes from here.

File: _posts/2016-01-01-jekmdl.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+date: 2016-01-01 22:51:45 07:00
+homepage: https://github.com/tigefa4u/jekmdl
+download: https://github.com/tigefa4u/jekmdl/archive/master.zip
+demo: http://tigefa4u.github.io/jekmdl/
+author: sugeng tigefa
+thumbnail: jekmdl.jpg
+license: MIT License
+license_link: https://github.com/tigefa4u/jekmdl/blob/master/LICENSE
+---
+# [jekmdl](https://github.com/tigefa4u/jekmdl)
```
Comment (gynter) [inline _posts/2016-01-01-jekmdl.markdown]: Please add an empty newline between front matter and content.

**Comment by gynter** [comment]
Please change the thumbnail image dimensions to 250x200.

Thanks in advance and Happy New Year!

**Comment by tigefa4u** [comment]
Happy new year too. 
all task done :sparkling_heart:

**Pull request merged**